lib/validators/form_template_yaml_validator.rb

```diff
@@ -1,39 +1,66 @@
 # frozen_string_literal: true
 
+RESERVED_KEYWORDS = %w[title body category category_id tags]
+
```

Comment:
Is there a reason this is defined both inside and outside the class? I think this definition can be removed right?

Reply:
Oh yes, this is a leftover, nice catch, thanks!